server/src/main/java/org/opensearch/common/unit/Fuzziness.java

```diff
@@ -233,6 +236,31 @@ private boolean isAutoWithCustomValues() {
         return fuzziness.startsWith("AUTO") && (lowDistance != DEFAULT_LOW_DISTANCE || highDistance != DEFAULT_HIGH_DISTANCE);
     }
 
+    private boolean isValidFuzzinessValue(String fuzziness) {
+        fuzziness = fuzziness.toUpperCase();
+        if("AUTO".equals(fuzziness)) {
+            return true;
+        }
+        if(fuzziness.startsWith("AUTO:")) {
+            String[] fuzzinessLimit = fuzziness.substring("AUTO:".length()).split(",");
+            return isValidFuzzinessLimit(fuzzinessLimit[0]) && isValidFuzzinessLimit(fuzzinessLimit[1]);
```

Comment:
You can get an exception there if input string has no comma (`,`).

Reply:
This is anyway covered in parsing. To make sure the utility validating fuzziness value is accurate, have added the check to this.